### DISCUSSION TITLE
Twilio Segment: the order of elements inside the modal component

### DISCUSSION THREAD

**Discussion by karoltonik** [twilio-labs/paste#3373] [Help: Figma] [answered]
I noticed that inside the modal component, the canvas stacking option in the auto layout is set to 'last on top.' This setting prevents creating custom content containing dropdown lists, as they appear under the buttons. Changing this option to 'first on top' would solve the issue.

[Figma link ](https://www.figma.com/file/oeUbF1qpn2FUsmXwiTsHmN/Segment-App%3A-Paste-Migration?type=design&node-id=1115%3A401467&mode=design&t=bAE3PKDNod0y9JWh-1)

**Comment by serifluous** [comment, marked as answer]
Just published a fix for this, thanks!

**Comment by karoltonik** [comment]
hey @serifluous, noticed the same problem as some time ago, has the previous version of 'last on top' been restored?
[figma preview](https://www.figma.com/file/oeUbF1qpn2FUsmXwiTsHmN/Segment-App%3A-Paste-Migration?type=design&node-id=1622%3A129377&mode=design&t=qaY2fKmBVxWNSE5q-1)

**Reply by serifluous** [reply]
Hey @karoltonik ! We haven't published any new changes to the Modal component since the update. Can you try pulling in component updates, or resetting changes on the component?
